### PR TITLE
[WIP] Add news to SR

### PR DESCRIPTION
**Pull request by fernandog**
@miigotu now it only get the changed.md

don't know how to fix the "mouse over" selection in inc_top

Also we need to decided the "layout" of this news page

### CODE REVIEW

File: sickbeard/webserve.py [hunk 1]
```diff
@@ -59,6 +59,7 @@
 from lib.trakt.exceptions import traktException
 from versionChecker import CheckVersion
 import babelfish
+from lib import requests
```
Comment (miigotu) [inline sickbeard/webserve.py]: dont from lib. just from requests. otherwise we have 2 objects.

**Comment by miigotu** [comment]
@fernandog I added a 'add-news' branch here, PR and merge it there so we both can work on it without putting in develop until it is done.

**Comment by fernandog** [comment]
What should I do? PR this to the new branch? Versionchecker.py has this from lib too

**Comment by miigotu** [comment]
Yes.
`Versionchecker.py has this from lib too`
Nope, you need to git pull then or idk, https://github.com/SiCKRAGETV/SickRage/blob/master/sickbeard/versionChecker.py#L36

**Comment by fernandog** [comment]
@miigotu done

https://github.com/SiCKRAGETV/SickRage/commits/add-news

**Pull request closed without merging**